package.json

```diff
@@ -55,6 +56,7 @@
   },
   "dependencies": {
     "@types/express": "^4.17.6",
+    "classnames": "^2.2.6",
```

Comment:
Is this an acceptable dependency to add? I'm a fan of using this cute little utility to make dynamic `className` generation a little smoother.

Reply:
For two callsites, probably not, given that it's like a single 60 LOC function, a trivial version could be added in the file, I've not quite got a utils folder, but maybe lib could do that (I agree that it's a clever abstraction, a nice declarative way to describe classes, here's a TS port of just the stuff you're using)

https://www.typescriptlang.org/play?#code/JAMwrgdgxgLglgewgAigDwBQDocEMBOA5gM4BcyGxM+cEhyAPspACYCmItbLjyIuAG2JsAlAG0AuiOQBvALAAoYMChIqqAbmLDiyALzJJAbkXIz5syAT4MqiOoL0EIZI+LT5Ci9-NwXGAEJHaTt4CDA2Ux9vKE1tNmIsAAcwYgALDGConwBfRWV8NhgwfBRYrR0sACsEWgwAImR6kWzkPIUou2IEATYsAQRCDFazdAakNnqAGj5BYWQAMgWmmAB3BGapkdRMeonp2aE2ReX6tY2Z6giTlbTCyZavZBEgA